gradle/libs.versions.toml

```diff
@@ -75,26 +76,26 @@ scarlet-message-adapter-moshi = { module = "com.tinder.scarlet:message-adapter-m
 scarlet = { module = "com.tinder.scarlet:scarlet", version.ref = "scarlet"}
 scarlet-stream-adapter-coroutines = { module = "com.tinder.scarlet:stream-adapter-coroutines", version.ref = "scarlet" }
 scarlet-websocket-okhttp = { module = "com.tinder.scarlet:websocket-okhttp", version.ref = "scarlet" }
-springboot-autoconfigure = { module = "org.springframework.boot:spring-boot-autoconfigure" }
-springboot-starter = { module = "org.springframework.boot:spring-boot-starter" }
-springboot-starter-actuator = { module = "org.springframework.boot:spring-boot-starter-actuator" }
-springboot-starter-aop = { module = "org.springframework.boot:spring-boot-starter-aop" }
-springboot-starter-jetty = { module = "org.springframework.boot:spring-boot-starter-jetty" }
-springboot-starter-web = { module = "org.springframework.boot:spring-boot-starter-web" }
+springboot-autoconfigure = { module = "org.springframework.boot:spring-boot-autoconfigure", version.ref = "springboot" }
+springboot-starter = { module = "org.springframework.boot:spring-boot-starter", version.ref = "springboot" }
+springboot-starter-actuator = { module = "org.springframework.boot:spring-boot-starter-actuator", version.ref = "springboot" }
+springboot-starter-aop = { module = "org.springframework.boot:spring-boot-starter-aop", version.ref = "springboot" }
+springboot-starter-jetty = { module = "org.springframework.boot:spring-boot-starter-jetty", version.ref = "springboot" }
+springboot-starter-web = { module = "org.springframework.boot:spring-boot-starter-web", version.ref = "springboot" }
 sqlite = { module = "org.xerial:sqlite-jdbc", version.ref = "sqlite" }
 
 # Test
 coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "kotlin-coroutines" }
 kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test", version.ref = "kotlin" }
 mockk = { module = "io.mockk:mockk", version.ref = "mockk" }
-springboot-starter-test = { module = "org.springframework.boot:spring-boot-starter-test" }
+springboot-starter-test = { module = "org.springframework.boot:spring-boot-starter-test", version.ref = "springboot" }
 testcontainers = { module = "org.testcontainers:testcontainers", version.ref = "testcontainers" }
 testcontainers-postgres = { module = "org.testcontainers:postgresql", version.ref = "testcontainers" }
 
 [bundles]
 database = ["exposed-core", "exposed-dao", "exposed-jdbc", "flyway", "hikari", "postgres", "sqlite"]
 eventstream = ["figure-eventstream-api", "figure-eventstream-api-model", "figure-eventstream-cli", "figure-eventstream-core"]
-grpc = ["grpc-netty", "grpc-netty-shaded", "grpc-protobuf", "grpc-stub", "grpc-springboot-starter", "grpc-kotlin-stub"]
+grpc = ["grpc-netty", "grpc-netty-shaded", "grpc-protobuf", "grpc-stub", "grpc-kotlin-stub"]
```

Comment:
The proto project brings in the grpc bundle, but doesn't need grpc-springboot-starter (and was actually not compiling with the latest version, probably needed to also bring springboot into that project which seemed... wrong)... So now I'm pulling in grpc-springboot-starter to the server project only.